### PR TITLE
Add ChefEffortless/Berksfile

### PR DESCRIPTION
**Pull request by tas50**
Identifies cookbooks that are still using Berkshelf for depsolving

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

File: config/cookstyle.yml [hunk 1]
```diff
@@ -920,6 +920,13 @@ ChefEffortless/SearchForEnvironmentsOrRoles:
   Exclude:
     - '**/metadata.rb'
 
+ChefEffortless/Berksfile:
+  Description: Policyfiles should be used for cookbook depsolving instead of a Berkshelf Berksfile.
```
Comment (afiune) [inline config/cookstyle.yml]: is `depsolving` an actual word? 🤔 cc/ @mjingle

Comment (tas50) [reply config/cookstyle.yml]: We use it in Cheflandia a lot

Comment (mjingle) [reply config/cookstyle.yml]: @afiune Supposedly it is, but it is an informal technical term. After doing a deep, deep dive search,`depsolving` supposedly means "dependencies solving." I would encourage using a less obscure, and less technical wording instead.

Comment (tas50) [reply config/cookstyle.yml]: Updated to dependency solving

**Pull request merged**